src/components/20-molecules/accordeon/index.js

```diff
@@ -0,0 +1,114 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { LitElement, html, css, unsafeCSS, svg } from 'lit-element';
+import expandLess from '@axa-ch/materials/icons/material-design/expand_less.svg';
+import expandMore from '@axa-ch/materials/icons/material-design/expand_more.svg';
+
+import { classMap } from 'lit-html/directives/class-map';
+import { defineVersioned } from '../../../utils/component-versioning';
+import { applyDefaults } from '../../../utils/with-react';
+import styles from './index.scss';
+
+class AXAAccordeon extends LitElement {
+  static get tagName() {
+    return 'axa-accordeon';
+  }
+
+  static get styles() {
+    return css`
+      ${unsafeCSS(styles)}
+    `;
+  }
+
+  static get properties() {
+    return {
+      small: { type: Boolean, defaultValue: false },
+      open: { type: Boolean },
+      disabled: { type: Boolean },
+      icon: { type: String },
+      title: { type: String },
+    };
+  }
+
+  constructor() {
+    super();
+    applyDefaults(this);
+  }
+
+  toggleAccordion() {
+    if (!this.disabled) {
+      this.open = !this.open;
+      this.toggleAnimation();
+    }
+  }
+
+  firstUpdated() {
+    this.toggleAnimation();
+  }
+
+  toggleAnimation() {
+    const accordeonContent = this.shadowRoot.querySelector(
+      '.m-accordeon__content'
+    );
+
+    if (this.open) {
+      accordeonContent.style.maxHeight = `${accordeonContent.scrollHeight}px`;
+    } else {
+      accordeonContent.style.maxHeight = 0;
```

Comment:
This seems unnecessary. Everything done in here could be made with CSS, no?

Reply:
Possibly there would be a way (I did not found one) but this solution is way more convenient:

1. You need JavaScript anyways and this method needs less code (Otherwise you have to add classes to the element and remove them)
2. It also needs less CSS Code since you do not need keyframes which makes the code way more readable.